scripts/security-checker.mjs

```diff
@@ -69,13 +69,15 @@ class SecurityChecker {
 
   createAlertDictionary (existedIssues) {
       return existedIssues.reduce((res, issue) => {
-          const [, repo] = issue.body.match(/Repository:\s*`(.*)`/);
-          const [, url, type, number] = issue.body.match(/Link:\s*(https:.*\/(dependabot|code-scanning)\/(\d+))/);
+          const [repo] = issue.body.match(/(?<=Repository:)[\s\S]*?(?=####|$)/g);
+          const [, url, type] = issue.body.match(/Link:\s*(https:.*\/(dependabot|code-scanning)\/(\d+))/);
+          const [, cveId] = issue.body.match(/CVE ID:\s*`(.*)`/);
+          const [, ghsaId] = issue.body.match(/GHSA ID:\s*`(.*)`/);
 
-          if (!url || repo !== this.context.repo)
+          if (!url)
               return res;
 
-          res[url] = { issue, number, type };
+          res[issue.title] = { issue, type, cveId, ghsaId, repo};
```

Comment:
If you want to use a title as a key, please use Map instead of a simple object, since the object has limitations for key names